src/main/groovy/io/openliberty/tools/gradle/tasks/InstallFeatureTask.groovy

```diff
@@ -37,10 +37,21 @@ class InstallFeatureTask extends AbstractFeatureTask {
 
     @TaskAction
     void installFeature() {
+        def propertiesList = InstallFeatureUtil.loadProperties(getInstallDir(project))
+        def openLibertyVersion = InstallFeatureUtil.getOpenLibertyVersion(propertiesList)
+
+        Boolean skipBetaInstallFeatureWarning = Boolean.parseBoolean(System.getProperty("skipBetaInstallFeatureWarning"))
```

Comment:
Same comments as https://github.com/OpenLiberty/ci.maven/pull/894#discussion_r516086013 and https://github.com/OpenLiberty/ci.maven/pull/894#discussion_r516087246